packages/calcite-components/src/components/action-bar/action-bar.tsx

```diff
@@ -45,7 +45,10 @@ export class ActionBar extends LitElement {
 
   private actionGroups: ActionGroup["el"][];
 
-  private mutationObserver = createObserver("mutation", () => this.mutationObserverHandler());
+  private mutationObserver = createObserver("mutation", () => {
+    this.mutationObserverHandler();
+    this.updateSlottedActions();
```

Comment:
Let's move this into `mutationObserverHandler`. 

Side note: I think we can also inline that function here since it's not used elsewhere.